add-no-translations.js

```diff
@@ -0,0 +1,68 @@
+const fs = require('fs');
```

Comment:
Can we move this to the scripts directory & ignore in .codeclimate.yml?

Reply:
I’m going to delete this file when I’m finished working on it. I open draft PRs as early as possible so this is not production ready yet! I’ll pop it into the code review column when I need keen eyes on it 🙂